Assets/Scripts/PlayerMovement/CatMovement.cs

```diff
@@ -388,10 +390,21 @@ IEnumerator SetObject()
         while (map == null)
         {
             map = GameObject.Find("Map");
-            //blockManager = FindObjectOfType<BlockManager>();
             yield return new WaitForEndOfFrame();
         }
         blockManager = map.GetComponentInChildren<BlockManager>();
+        if(map.GetComponentInChildren<StartingBlock>() == null)
+        {
+            //it doesn't exist
+            Debug.Log("starting block not made");
+            transform.position = Vector3.zero;
+        }
+        else
+        {
+
+        }
+            transform.position = map.GetComponentInChildren<StartingBlock>().GetPos();
```

Comment:
? shouldn't this be in the else statement?

Reply:
oh right, it should be haha